django_cloud_deploy/workflow/__init__.py

```diff
@@ -151,9 +169,24 @@ def create_and_deploy_new_project(self,
             self._generate_section_header(
                 7, 'Create Service Account Necessary For Deployment',
                 self._TOTAL_NEW_STEPS))
-        secrets = self._service_account_workflow.handle_service_accounts(
-            project_id, database_username, database_password
-        )
+
+        # Generate required Kubernetes secrets with service account keys.
+        secrets = {
```

Comment:
can we put all of this logic into one function? generate_secrets?

Reply:
Done.